extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -666,49 +580,59 @@ export class SqlDatabaseTree extends AssessmentDialogComponent {
 					]
 				]
 			}
-		);
-
-		this._assessmentResultsTable.component().onRowSelected(({ row }) => {
-			this._descriptionText.value = this._assessmentResultsTable.component().data![row][0];
-
-			if (this._dbName.value) {
-				this._issues = this._assessmentData.get(this._dbName.value)![row];
-				this._moreInfo.value = this._issues.moreInfo;
-				this._impactedObjects = this._issues.impactedObjects;
-				let data: { value: string; }[][] = [];
-				this._impactedObjects.forEach(async (impactedObject) => {
-					data.push([
-						{
-							value: impactedObject.objectType
-						},
-						{
-							value: impactedObject.name
-						}
-
-					]);
-				});
-
-				this._assessmentTitle.value = this._issues.description;
-
-				this._impactedObjectsTable.component().updateProperties({
-					dataValues: data
-				});
+		).component();
+
+		this._assessmentResultsTable.onRowSelected(({ row }) => {
+			this._selectedIssue = this._activeIssues[row];
+			this._assessmentTitle.value = this._selectedIssue.checkId;
+			this._descriptionText.value = this._selectedIssue.description;
+			this._moreInfo.url = this._selectedIssue.helpLink;
+			this._moreInfo.label = this._selectedIssue.helpLink;
+			this._impactedObjects = this._selectedIssue.impactedObjects;
+			this._impactedObjectsTable.dataValues = this._selectedIssue.impactedObjects.map((object) => {
+				return [
+					{
+						value: object.objectType
+					},
+					{
+						value: object.name
+					}
+				];
+			});
+			if (this._impactedObjects.length > 0) {
+				this._objectDetailsName.value = this._impactedObjects[0].name;
+				this._objectDetailsType.value = this._impactedObjects[0].impactDetail;
 			}
+
 		});
 
-		return this._assessmentResultsTable.component();
+		return this._assessmentResultsTable;
 	}
 
 	public selectedDbs(): string[] {
 		let result: string[] = [];
-
-		this.databaseTable.component().dataValues?.forEach((arr) => {
+		this._databaseTable.dataValues?.forEach((arr) => {
 			if (arr[0].value === true) {
 				result.push(arr[1].value.toString());
 			}
 		});
-
 		return result;
 	}
 
+	public selectDbs(dbNames: string[]): void {
```

Comment:
A comment here would be using - what is this "selecting"? It just seems to be updating the table values